org.lflang/src/org/lflang/generator/FederateInstance.xtend

```diff
@@ -369,6 +369,7 @@ class FederateInstance {
     
     /** The generator using this. */
     var generator = null as GeneratorBase
+    def getGenerator() { return generator; }
```

Comment:
Undocumented function.

Reply:
Should getters and setter be documented? The document would just say `Returns generator`.

Reply:
It's not a big deal. Active annotation can be used for this type of thing in Xtend. Perhaps using that will generate better documentation automatically. It looks something like:
```
 @Accessors(PUBLIC_GETTER, PROTECTED_SETTER) int age
```

Reply:
I think using this type of annotation requires adding one more dependency: `org.eclipse.xtend.lib.annotations`. 
I will just add a comment here, which I think will be less painful when we port this class to Kotlin or Java.